tools/pip/setup.py

```diff
@@ -215,4 +217,4 @@ def skip_markdown_comments(md):
           'Topic :: Software Development :: Libraries',
           'Topic :: Software Development :: Libraries :: Python Modules',
       ],
-      url='https://github.com/apache/incubator-mxnet')
+      url='https://github.com/apache/incubator-mxnet')
```

Comment:
nitpick: newline at the end of the file.

Reply:
New line added